modules/reissue-internal-certificates-secured-clusters.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * configuration/reissue-internal-certificates.adoc
+:_module-type: PROCEDURE
+[id="reissue-internal-certificates-secured-cluster_{context}"]
+= Reissuing internal certificates for Secured Clusters using init bundles
+
+Secured clusters contain the Collector, Sensor, and Admission Control components. These components use a built-in server certificate for authentication when communicating with other {product-title} components.
+
+The RHACS portal shows an information banner when the Central certificate is about to expire.
+
+[NOTE]
+====
+The information banner only appears 15 days before the certificate expiry date.
+====
+
+.Prerequisites
+
+* To reissue certificates, you must have `write` permission for the `ServiceIdentity` resource.
+
+[IMPORTANT]
+====
+Store this bundle securely because it contains secrets.
+You can use the same bundle on multiple secured clusters.
+====
+
+.Procedure
+
+To generate an init bundle using the {product-title-short} portal:
+
+. Choose *Platform Configuration* -> *Clusters*.
+. Click *Manage Tokens*.
+. Navigate to the *Authentication Tokens* section and click on *Cluster Init Bundle*.
+. Click *Generate bundle*.
+. Enter a name for the cluster init bundle and click *Generate*.
+. Click *Download Kubernetes secrets file* to download the generated bundle.
+
+To generate an init bundle using the `roxctl` CLI, enter the following command:
+
+[source,terminal]
+----
+$ roxctl -e <endpoint> -p <admin_password> central init-bundle generate <bundle_name> --output-secrets init-bundle.yaml
+----
+
+.Next steps
+* Use the `oc -n stackrox apply -f <init-bundle.yaml>` command to create the necessary resources on each secured cluster.
```

Comment:
```suggestion
* To create the necessary resources on each secured cluster, run the following command:
[source,terminal]
oc -n stackrox apply -f <init-bundle.yaml>
```
Per the OpenShift Docs guidance on [Code blocks, command syntax, and example output](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output):

> - If a step in a procedure is to run a command, make sure that the step text includes an explicit instruction to "run" or "enter" the command. In most cases, use one of the following patterns to introduce the code block:
>  _Step description_, run the following command:
>  _Step description_, enter the following command:
> 
> - Do NOT use any markup in code blocks; code blocks generally do not accept any markup.